docs/changelog/95481.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 95481
+summary: "Add support for `logs@custom` component template for `logs-*-* data streams"
```

Comment:
Worth mentioning the removal bug fix you did here to? I don't know if you can have more then one changelog per PR. If not, likely needs to be taken out.